vite.config.ts

```diff
@@ -0,0 +1,46 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import eslint from 'vite-plugin-eslint';
+
+const largeLibraries = [
+  '@mui',
+  '@uppy',
+  '@aws-amplify',
+  'imask',
+  'i18next',
+  'date-fns',
+  '@tanstack',
+  'isbn3',
+  'yup',
+  'axios',
+  'formik',
+];
+
+export default defineConfig({
+  plugins: [react(), eslint()],
+  server: {
+    open: true,
+    port: 3000,
+  },
+  preview: {
+    port: 3000,
+  },
+  build: {
+    outDir: 'build',
+    rollupOptions: {
+      onwarn: ({ message }) => {
+        if (message.startsWith('[plugin vite-plugin-eslint]')) {
+          throw new Error(message);
+        }
+      },
+      output: {
```

Comment:
Har hatt en lang brytekamp over flere omganger med byggesteget her. Får fortsatt noen warnings av at noen chunks er store, samtidig som det lages en del som er bittesmå. Har mitigert begge deler med hhv `experimentalMinChunkSize` og `manualChunks`, så det er mye bedre enn var i utgangspunktet. Tror ikke det er verdt å gjøre mer tilpasninger enn dette, så tenker dette er OK. Ikke helt sikker på om `largeLibraries`-løsningen er bare positiv heller, men it's the best i got 🤔

 Grunnen til at dette er litt knot er at vi bruker code splitting for å unngå at hele appen lastes med en gang, vi laster bare det brukeren trenger der man er, kort sagt. Rollup (som brukes i bygget hos Vite) flytter derfor mange ting til sine egne små chunks, som ikke importeres før brukeren går til en side hvor den komponenten/kodesnutten brukes. Dette fører til at nettleseren ved første lasting kan måtte risikere å laste ned 30+ små chunks, som vi egentlig er ganske sikre på at man trenger.

Jeg har ikke veldig mye peiling på dette, og hva som er best-practice, og det blir en del prøving og feiling. Men jeg tror det som er her nå er bra nok til å tas i bruk